build/bin/sage-bootstrap-python

```diff
@@ -60,7 +58,7 @@ PYTHONS="python python3 python3.12 python3.11 python3.10 python3.9 python3.8 pyt
 for PY in $PYTHONS; do
     PYTHON="$(PATH="$SAGE_ORIG_PATH" command -v $PY)"
     if [ -n "$PYTHON" ]; then
-        if CHECK_PATH="$PATH" "$PYTHON" -c "import argparse; import urllib; from hashlib import sha1; from ssl import HAS_SNI; assert HAS_SNI; from os import listdir, environ; listdir(\"$(pwd)\"); assert environ[\"PATH\"] == environ[\"CHECK_PATH\"];" 2>/dev/null; then
```

Comment:
It's likely that this is also helpful on WSL, but I can't test it now.

Reply:
I don't think WSL is operating on naked Windows paths.

Reply:
No, but this is also not what is tested here.

Reply:
Sorry, I misread what is does. In fact, it's something to do with Pythons which have crippled ssl module. On WSL Python is just a normal modern Ubuntu, or whatever Linux distro it uses, Python.

Reply:
There is a test does check for ssl, but the part removed above checks for something else/

And on WSL it *is* possible to get some Python.exe through the PATH.

Reply:
`Python.exe` is not `python3`, so I can't imagine this being a problem.

Reply:
I've restored it in 52a657ab127, with an updated comment.